flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -127,13 +128,29 @@
     private boolean forceAvro = false;
     private long autoWatermarkInterval = 200;
 
+    // ---------- statebackend related configurations ------------------------------
     /**
      * Interval in milliseconds for sending latency tracking marks from the sources to the sinks.
      */
     private long latencyTrackingInterval = MetricOptions.LATENCY_INTERVAL.defaultValue();
 
     private boolean isLatencyTrackingConfigured = false;
 
+    /** Interval in milliseconds to perform periodic materialization. */
+    private long periodicMaterializeInterval =
+            StateBackendOptions.PERIODIC_MATERIALIZATION_INTERVAL.defaultValue();
+
+    /** Interval in milliseconds for initial delay of periodic materialization. */
+    private long periodicMaterializeInitDelay =
+            StateBackendOptions.PERIODIC_MATERIALIZATION_INIT_DELAY.defaultValue();
```

Comment:
Why `periodicMaterializeInterval` isn't sufficient?

Reply:
That's the init delay. I can remove it if you think it is not necessary. I do not have a strong opinion.

Reply:
I think the interval should be enough (one less option to learn for users).